docs/static/settings/configuration-management-settings.asciidoc

```diff
@@ -54,9 +54,10 @@ section in your Logstash configuration, or a different one. Defaults to
 If your {es} cluster is protected with basic authentication, these settings
 provide the username and password that the Logstash instance uses to
 authenticate for accessing the configuration data. The username you specify here
-should have the built-in `logstash_admin` role and the customized `logstash_writer` role, which provides access to system
-indices for managing configurations. Starting with Elasticsearch version 7.10.0, the
-`logstash_admin` role inherits the `manage_logstash_pipelines` cluster privilege for centralized pipeline management.
+should have the built-in `logstash_admin` role and `logstash_system` role, which provides access to system
+indices for managing configurations.
```

Comment:
```suggestion
should have the built-in `logstash_admin` role and `logstash_system` role.
These roles provide access to system indices for managing configurations.
```

Reply:
You don't have to break this into two sentences if you're not feeling it. 
However, `provides` should be `provide`.